src/lammpsio/snapshot.py

```diff
@@ -437,6 +442,114 @@ def has_mass(self):
         """
         return self._mass is not None
 
+    @property
+    def bonds(self):
+        """:class:Bonds: bonds."""
+        if not self.has_bonds():
+            self._bonds = None
```

Comment:
This doesn't need to be here. If `has_bonds() is False`, `self._bonds is None`.
```suggestion
```
Please repeat for all of them.